config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
   root controller: :main, action: :index
   get '*path', to: 'main#index', via: :all
```

Comment:
What does via: :all mean?

Reply:
Basically every time we would go to to another route like '/register' we would go to '/' instead, this way we can use the router in react to render the component we want at that url.